### ISSUE TITLE
Legacy DSA signing fails

### ISSUE THREAD

**Issue by Daeinar** [Legrandin/pycryptodome#316]
Consider the following example `dsa.py`:

```dsa.py
from Crypto.PublicKey import DSA
from Crypto.Signature import DSS

if __name__ == '__main__':
    key = DSA.generate(1024)
    dsa = DSS.new(key,'fips-186-3')
```

Running `python3 dsa.py` it crashes with the following error:

```
Traceback (most recent call last):
  File "dsal.py", line 6, in <module>
    dsa = DSS.new(key,'fips-186-3')
  File "/usr/local/lib/python3.7/site-packages/Crypto/Signature/DSS.py", line 406, in new
    print(key)
  File "/usr/local/lib/python3.7/site-packages/Crypto/PublicKey/DSA.py", line 193, in __repr__
    attrs.append("p(%d)" % (self.size()+1,))
  File "/usr/local/lib/python3.7/site-packages/Crypto/PublicKey/DSA.py", line 356, in size
    raise NotImplementedError
NotImplementedError
```

This can be traced back to: https://github.com/Legrandin/pycryptodome/blob/master/lib/Crypto/PublicKey/DSA.py#L356

Is legacy DSA deprecated? The [official documentation](https://www.pycryptodome.org/en/latest/src/signature/dsa.html#module-Crypto.Signature.DSS) suggests that it should be still supported via `Crypto.Signature.DSS`.

**Comment by Legrandin** [comment]
Thanks for reporting. There is no `print(key)` statement in DSS.py: probably you added it to debug something else? Without that, DSA signature should work.

In general, it `print(key)` should work too nonetheless: I will fix that...

**Comment by Daeinar** [comment]
Yeah right, I forgot the `print(key)` statement while trying to find out how the library works. Thanks!

**Comment by Legrandin** [comment]
Fixed with 4dbbf65901a23dd0fbdce1512e1ae71e673670e3
